test/test_reg.py

```diff
@@ -1,21 +1,19 @@
-"""  This page is the page where pytest detects automatically and
-    through which all the methods or APIs are called through  """
-
-from config import *
+import pytest
 from pom.irctc_pom import Register
 from pom.flight_pom import FlightRegister
-from Library.basefixture import Driver_init
+from config import url

```

Comment:
All these import shouldn't present in test case.
Everything should come through conftest file.